exercise/exercise_summary.py

```diff
@@ -36,27 +39,46 @@ def __init__(self, exercise, user=None):
                     Submission.STATUS.REJECTED,
                 ):
                     self.submission_count += 1
-                    if (
-                        s.status == Submission.STATUS.READY and (
-                            self.best_submission is None
-                            or self.unofficial
-                            or s.grade > self.best_submission.grade
-                        )
-                    ):
+
+                    if s.force_exercise_points:
+                        # This submission is chosen as the best submission and
+                        # no further submissions are considered.
                         self.best_submission = s
                         self.unofficial = False
```

Comment:
Like in exercise/cache/points.py, what if an unofficial submission is marked as the final graded submission in the manual assesment UI?

Reply:
See previous comments, submissions are now set to status 'ready' when they are marked as final.